src/ImmediateContext.cpp

```diff
@@ -545,16 +546,13 @@ void ImmediateContext::RollOverHeap(OnlineDescriptorHeap& Heap) noexcept(false)
     }
     else
     {
-        if constexpr (g_bUseRingBufferDescriptorHeaps)

```

Comment:
This check isn't in the original.  Should it be there too?

Reply:
It absolutely should've been, since we were getting some telemetry spam, but that version isn't taking new updates.